File: Factory/SimpleFactory/TypeScript/src/PizzaFriday.ts
import SimplePizzaFactory from './SimplePizzaFactory';
import PizzaStore from './PizzaStore';

const pizzaFriday = () => {
  const factory = new SimplePizzaFactory();
  const store = new PizzaStore(factory);

  let pizza = store.orderPizza('cheese');
  console.log(pizza.toString());

  pizza = store.orderPizza('pepperoni');
  console.log(pizza.toString());
};

pizzaFriday();

File: Factory/SimpleFactory/Java/src/PizzaFriday.java
package eu.svetlin.design.patterns.factory.simplefactory;

import eu.svetlin.design.patterns.factory.simplefactory.Pizza.Pizza;

public class PizzaFriday {
   public static void main(String[] args){
        SimplePizzaFactory factory = new SimplePizzaFactory();
        PizzaStore store = new PizzaStore(factory);

        Pizza pizza = store.orderPizza("cheese");
        System.out.println(pizza);

        pizza = store.orderPizza("pepperoni");
        System.out.println(pizza);
    }
}
